src/Symfony/Bundle/WebServerBundle/Command/ServerStartCommand.php

```diff
@@ -141,8 +142,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
 
             $config = new WebServerConfig($documentRoot, $env, $input->getArgument('addressport'), $input->getOption('router'));
+            $isStarted = WebServer::STARTED === $server->start($config, $input->getOption('pidfile'));
+            if ($isStarted && 'json' === $input->getOption('format')) {
+                printf(json_encode(array('hostname' => $config->getHostname(), 'port' => $config->getPort())));
```

Comment:
Why not use the `OutputInterface::write(ln)` here? And you should be able to use short array syntax here?

Reply:
we don't use short array syntax in the Symfony code base, neither on 3.* nor 4.*